vendorconsent/consent.go

```diff
@@ -1,48 +1,47 @@
 package vendorconsent
 
 import (
-	"encoding/base64"
 	"fmt"
-	"strings"
 
 	"github.com/prebid/go-gdpr/api"
 	tcf1 "github.com/prebid/go-gdpr/vendorconsent/tcf1"
 	tcf2 "github.com/prebid/go-gdpr/vendorconsent/tcf2"
 )
 
+var (
+	errEmptyDecodedConsent = fmt.Errorf("decoded consent cannot be empty")
```

Comment:
We use this in several places. Perhaps we want to consider exporting this from consentconstants, which would have the side effect of allowing the user of the library to do a simple compare to check that an error is this error rather than doing string matching.

Reply:
Oooo.. I like that.

Reply:
There's no need to use `fmt.Errorf`. Since it's a static message, `errors.New` would be a better choice.